rllib/algorithms/dreamerv3/README.md

```diff
@@ -1,27 +1,94 @@
-interactions. The world model utilizes a recurrent GRU-based architecture
-("recurrent state space model" or RSSM) and uses it to predicts rewards,
-episode continuation flags, as well as, observations.
-With these predictions (dreams) made by the world model, both actor
-and critic are trained in classic REINFORCE fashion. In other words, the
-actual RL components of the model are never trained on actual environment data,
-but on dreamed trajectories only.
+## Overview
+An RLlib-based implementation for TensorFlow/Keras of the "DreamerV3" model-based reinforcement
+learning algorithm by D. Hafner et al. (Google DeepMind) 2023
 
-For more algorithm details, see:
+The implementation is based on RLlib's new Learner- and RLModule APIs and therefore allows
+for utilizing multi-GPU machines for training (see below for tips and tricks, example configs,
+and command lines).
+
+DreamerV3 trains a world model in supervised fashion using real environment
+interactions. The world model's objective is to correctly predict all aspects
+of the transition dynamics of the RL environment, which includes (besides predicting the
+correct next observations) predicting the received rewards as well as a boolean episode
+continuation flag.  
+It utilizes a recurrent GRU-based architecture called "recurrent state space model" or RSSM.
+Alternatingly to training this world model, it is used to generate (or "dream") ficticious
+trajectories, with which two further models are trained in an RL-fashion:
+The critic and actor networks.
+Just like in a standard policy gradient algorithm (e.g. REINFORCE), the critic tries to
+predict a correct value function (based on the world model-predicted rewards), whereas
+the actor tries to come up with good actions to take for maximizing accumulated rewards
+over time.
+In other words, the actual RL components of the model (actor and critic) are never
+trained on real environment data, but on dreamed trajectories only.
+
+## Note on Hyperparameter Tuning for DreamerV3
+DreamerV3 is an extremely versatile and stable algorithm that not only works well on
+different action- and observation spaces, including discrete and continuous actions, as well
+as image and vector observations, but also almost needs no hyperparameter tuning, except for
+a simple "model size" setting (from "XS" to "XL") and a value for the training ratio, which
+specifies how many steps to replay from the buffer for a training update vs how many
+steps to take in the actual environment.
+
+
+## Note on multi-GPU Training with DreamerV3
+We found that when using multiple GPUs for DreamerV3 training, the following simple
+adjustments should be made to the standard config.
+- Multiply the original batch size (B=16) by the number of GPUs you are using.
+- Multiply the number of environments you sample from in parallel by the number of GPUs you are using.
+- Use a learning rate schedule for all learning rates (world model, actor, critic) with "priming".
```

Comment:
Can you link `priming` to the relevant arxiv link for the papers here?

Reply:
done